src/hosting-api.test.js

```diff
@@ -1,35 +1,39 @@
 "use strict";
 
 import hosting from "./hosting-node.js";
-import nock from "nock";
+import https from "https";
 /* eslint-disable jest/no-disabled-tests */
+
+process.env.CO2JS_VERSION = "1.2.34";
+const requestHeaderComment = "TestRunner";
+
 describe("hostingAPI", () => {
+  let httpsGetSpy;
+  beforeEach(() => {
+    httpsGetSpy = jest.spyOn(https, "get");
```

Comment:
AsI understand it, this would move our project from using a _mock_, which doesn't actually make network requests, to using a _spy_,  [which still calls the underlying function](https://jestjs.io/docs/jest-object#jestspyonobject-methodname), resulting in a network call.

If we're already not making network requests, I'd prefer we not get into the habit of making network requests in test suites if possible. However, I get that this is something we can do separately from this PR. If we're not going to add an [mockimplementation](https://jestjs.io/docs/mock-function-api#mockfnmockimplementationfn), in this test, can we make an issue to come back to this, so there's a nice well defined issue for someone who is familiar with jest and mocking to pick up in future?

Reply:
Made those changes. Mocked `fetch` and `https.get` API requests.